detoks/src/main/res/values/string_arrays.xml

```diff
@@ -4,5 +4,10 @@
         <item>Random</item>
         <item>Highest watch time</item>
         <item>Lowest watch time</item>
```

Comment:
I would move these to be at the bottom, since they're less interesting, but less important obviously